buildconfig/download_win_prebuilt.py

```
@@ -86,8 +86,8 @@ def get_urls(x86=True, x64=True):
         'e3c79c1bde7a4b2064d1cd660d357e82a7f7e856'
         ],
         [
-        'https://github.com/pygame-community/SDL_image/releases/download/2.8.8-pgce/SDL2_image-devel-2.8.8-VCpgce.zip',
-        '8cd149cbaae2d362f2eaa69483d992e79f4836f5'
+        'https://github.com/pygame-community/SDL_image/releases/download/2.8.10-pgce/SDL2_image-devel-2.8.10-VCpgce.zip',
```

Flake8 (7.3.0)

[error] continuation line missing indentation or outdented

(E122)